Sell/Product/Tests/ProductModifierItemTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ProductModifierItemTests.swift
+//  RealifeTechTests
+//
+//  Created by Mickey Lee on 12/08/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import XCTest
+import GraphQL
+@testable import RealifeTech
+
+final class ProductModifierItemTests: XCTestCase {
+
+    func test_initWithFragment_nonNilResponse_returnsValidValue() throws {
+        let fragment = ApolloType.FragmentProductModifierItem(
```

Comment:
Do we have to reference Apollo?
Architecturally we are coupling the business logic to the framework, which is bad practice.